src/main/js/wiki/components/PageHeader.tsx

```diff
@@ -178,17 +183,29 @@ class PageHeader extends React.Component<Props, State> {
       />
     );
 
+    const branch_entries = [];
+    for (const branch of this.props.branches) {
+      branch_entries.push({ name: branch.name, link: branch._links.self.href });
+    }
+
+    const branchDropdown = (
+      <BranchDropdown
+        repository={this.props.wiki.repository}
+        currentBranch={this.props.currentBranch}
```

Comment:
`currentBranch` is no field in Props. Typescript error TS2339.

Reply:
introduced types